.github/workflows/tests.yml

```diff
@@ -1,10 +1,5 @@
 name: Tests
 
-# The wheel package of jpy is not reliable and causes
```

Comment:
Why do you remove this?

Reply:
@Alex-Izquierdo On Python 3.8 I was getting this failure
https://github.com/ansible/ansible-rulebook/actions/runs/3473072376/jobs/5804676382
Which says no not have that the pip_no_binary

Reply:
It was you who determined that the precompiled version of JPY is not reliable. Can we consider it reliable here? or shall we investigate why the compilation fails and fix the workflow accordingly?

Reply:
I did a quick test with python3.8 and it compiles fine, so it must be some issue only with the GH worker.

Reply:
The `ubuntu-latest` runner still points to ubuntu-20.04. Might be worth updating the workflow to force it to use `ubuntu-22.04`?

Reply:
Good point, that could solves the issue!

Reply:
@Alex-Izquierdo The JPY team has been making changes to improve the packaging when I started a while back there were issues and I had to ensure that we compile it locally instead of using the wheel directly. Given all the changes we could switch to using the wheel packages if they exist otherwise it would get compiled naturally.